docs/development/heat_dissipation.md

```diff
@@ -1,11 +1,92 @@
-# Heat dissipation
-
-::: warning TODO
-- Description of losses in power electronics
-    - Switching losses
-    - Conduction losses
-- Methods for cooling
-    - Passive using PCB
-    - Passive via heat sink
-    - Active (forced convection)
-:::
+# Losses and Heat
+
+In every real electrical system, flowing currents are generating heat inside the components. While most low energy devices do not generate significant amount of losses, power electronics need to dissipate large amount of thermal energy.
+
+There are many different components on a board that contribute to the overall power loss, like resistors, traces, fuses, coil resistance and other ohmic resistances. In circuits with frequent switching, the impedance of capacitators and inductances can play a role as well, but won't be treated here.
+
+The main factor for heat generation in power electronics are the semi-conductors, mainly **MOSFETs** used for switching. In the following, switching and conduction losses for MOSFETs are explained and different ways to dissipate the heat are given.
+
+## Conduction loss
+
+Conduction loss appears in the steady-state phase when the system is not changing anymore after being switched on. The power can be calculated using
+
+$$ Q_{cond} = R_{DS(on)} \cdot I^2 $$
+
+where $R_{DS}$ is the internal gate to source resistance of the transistor, which can be obtained from the datasheet and $I$ is the current flowing through it. If the transitor is used in converters or else and is switched on and off frequently (e.g. a PWM [charge controller](../system/charge_controller.md)), the duty cycle (in percent) should be used to weigh the power loss:
+
+$$ P_{cond} = R_{DS} \cdot I^2 \cdot D$$
+
+To reduce conduction loss, a bigger transistor with a larger channel can be used to reduce $R_{DS}$. That also means a longer switching time because the internal gate capacitance in the transistor is greater, which increases the switching time and therefore the switching loss. In high frequency switching applications with a low duty cycle, it can be better to optimize for low switching time than low internal resistance.
+
+## Switching loss
+
+When a transistor is switching, the conducting channel has to build up. When the gate to source voltage is higher than the threshold voltage, the channels starts to form and the resistance $R_{DS}$ starts to fall. Until the channel has formed completly, the transistor acts as a ohmic resistance, the power loss in it can be obtained by
+
+$$P_{swit} = \frac{1}{2\cdot T_s} \cdot t_{on} \cdot U_{ds} \cdot I_{ds}$$
```

Comment:
All heat flows should be `Q` as mentioned before (not `P`).

Also please use capital DS for drain-source voltage and current so that it's the same as for the resistor.

Final remark I realized just now: In English, `V` is the typical symbol for voltages and not `U`. In order to be consistent with datasheets of MOSFETs (see e.g. [here](https://www.mouser.de/datasheet/2/196/Infineon_IRFP4568_DataSheet_v01_01_EN-1228352.pdf)) I'd ask you to update this aswell.

Reply:
Ooops, sorry. The heat flow is actually the derivative of Q, so it must be Q with a dot. Seems like [this page](https://en.wikipedia.org/wiki/Thermal_resistance) on Wikipeda is wrong, as it uses Q without a dot all the time...

Reply:
Wikipedia page is now fixed. `\dot{Q}` is correct for heat flow.